tsconfig.base.json

```diff
@@ -26,9 +26,6 @@
       "@amplication/data-service-generator": [
         "libs/data-service-generator/src/index.ts"
       ],
-      "@amplication/data-service-generator/*": [
```

Comment:
why did you remove it?

Reply:
This change blocks us from making the same mistake again

Reply:
Why don't we need data-service-generator in the ts config?

Reply:
We need it, but the main entry only, the index.ts